sklearn/feature_extraction/tests/test_text.py

```diff
@@ -1105,7 +1105,7 @@ def test_tfidf_vectorizer_type(vectorizer_dtype, output_dtype,
     expected_warning_cls = warning_cls if warning_expected else None
     with pytest.warns(expected_warning_cls,
                       match=warning_msg_match) as record:
-            X_idf = vectorizer.fit_transform(X)
+        X_idf = vectorizer.fit_transform(X)
```

Comment:
@jnothman - Can you please check this line in particular  . It was giving me a flake8 error so had to touch it to pass the tests.